model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -1205,58 +1207,58 @@ message FunctionSpec {
   bytes payload = 3;
 }
 
-// TODO: transfer javadoc here
-message DisplayData {
-
-  // (Required) The list of display data.
-  repeated Item items = 1;
-
-  // A complete identifier for a DisplayData.Item
-  message Identifier {
-
-    // (Required) The transform originating this display data.
-    string transform_id = 1;
-
-    // (Optional) The URN indicating the type of the originating transform,
-    // if there is one.
-    string transform_urn = 2;
-
-    string key = 3;
+// A set of well known URNs describing display data.
+//
+// All descriptions must contain how the value should be classified and how it
+// is encoded. Note that some types are logical types which convey contextual
+// information about the pipeline in addition to an encoding while others only
+// specify the encoding itself.
+message StandardDisplayData {
+  enum DisplayData {
+    // A string label and value. Has a payload containing an encoded
+    // LabelledStringPayload.
+    LABELLED_STRING = 0 [(beam_urn) = "beam:display_data:labelled_string:v1"];
```

Comment:
Isn't it best practice to start with 1, so that unset == 0?

Reply:
It doesn't matter for these constant fields since we don't expect to use the enum as an actual field anywhere on a message and only use them for lookups. We can always change it if we need to in the future and it won't impact execution or existing pipelines either.